### ISSUE TITLE
Move Moment of Body from SSP unitals to physics.hs(concepts/quantities)

### ISSUE THREAD

**Issue by oluowoj** [JacquesCarette/Drasil#1911]
@smiths and @bmaclach 

Just investigating issue log #1901 with some of the examples and found Moment is defined only in SSP.
Do you agree that Moment needs to be defined globally? Its currently in SSP unitals and I see a comment there about this. I think we should move it to Physics domain but just wanted to run this by you both, unless there is a reason why @bmaclach wants to keep it there.

**Comment by bmaclach** [comment]
@oluowoj I agree, it's not SSP-specific and should be defined with the physics concepts. I can't think of any reason why we would need to keep it in SSP.

**Comment by smiths** [comment]
@oluowoj, I agree with you and with @bmaclach.  Moment is a very standard physics concept.  It will definitely be reused in other projects in the future.

**Comment by oluowoj** [comment]
@bmaclach, is it okay if I define it globally, then you can update SSP, wherever its being used?

**Comment by bmaclach** [comment]
@oluowoj Sure, sounds good.

**Comment by oluowoj** [comment]
PR #1913

**Comment by oluowoj** [comment]
PR #1913 has been merged, @bmaclach, you can go ahead and update SSP. Closing this issue.
